src/services/oidc-data.service.ts

```diff
@@ -33,7 +33,7 @@ export class OidcDataService {
         headers = headers.set('Accept', 'application/json');
 
         return this.httpClient.get<T>(url, {
-            headers: headers
+            headers: headers,
```

Comment:
comma too many?

Reply:
no because of git code styling --> use trailing commas